warehouse/static/js/warehouse/index.js

```diff
@@ -173,3 +173,16 @@ application.load(definitionsFromContext(context));
 docReady(() => {
   RepositoryInfo();
 });
+
+docReady(() => {
+  // browser warning for IE 10
+  if (navigator.appVersion.includes("MSIE 10")) {
+    var warning_html = "<div class='notification-bar notification-bar--danger'><span class='notification-bar__icon'><i class='fa fa-exclamation-triangle' aria-hidden='true'></i><span class='sr-only'>Warning:</span></span><span class='notification-bar__message'>You are using an unsupported browser, please upgrade to a newer version.</span></div>";
+
+    var warning_div = document.createElement("div");
+    warning_div.innerHTML = warning_html;
+
+    var warning_section = document.getElementById("sticky-top");
```

Comment:
note the element ID here is not correct, should be `stick-to-top` as defined below